package/linux-firmware/linux-firmware.mk

```diff
@@ -537,7 +537,7 @@ LINUX_FIRMWARE_IWL8000_UCODE_API_MAX = 36
 LINUX_FIRMWARE_IWL8265_UCODE_API_MAX = 36
 LINUX_FIRMWARE_IWL9000_UCODE_API_MAX = 46
 LINUX_FIRMWARE_IWL_22000_UCODE_API_MAX = 77
-LINUX_FIRMWARE_AX210_UCODE_API_MAX = 83
+LINUX_FIRMWARE_AX210_UCODE_API_MAX = 89
```

Comment:
This won't work - this variable tells what latest ucode version is supported by the current Linux kernel, and gets installed afterward. It also needs to be double checked with the used linux-firmware version, as the version must be available there as well. This change will break both the constraints - the newest ucode in linux-firmware 20240115 is 86, and even if you updated to that one, the kernel only supports up to 83, so it will attempt to load that one and fail.